src/components/Coaching/CoachingDetail/ActivitySummary/ActivitySummary.test.tsx

```diff
@@ -1,158 +1,117 @@
-
-    const thankYouRow = rows[9];
-    expect(thankYouRow.children[0]).toHaveTextContent('Thank Yous');
-    expect(thankYouRow.children[1]).toHaveTextContent('6');
-    expect(thankYouRow.children[2]).toHaveTextContent('16');
-    expect(thankYouRow.children[3]).toHaveTextContent('36');
-    expect(thankYouRow.children[4]).toHaveTextContent('19');
+    expect(initiationSocialMediaRow.children[1]).toHaveTextContent('34');
+    expect(initiationSocialMediaRow.children[2]).toHaveTextContent('14');
+    expect(initiationSocialMediaRow.children[3]).toHaveTextContent('4');
+    expect(initiationSocialMediaRow.children[4]).toHaveTextContent('4');
+
+    const inPersonAppointmentsRow = rows[11];
+    expect(inPersonAppointmentsRow.children[0]).toHaveTextContent('In Person');
+    expect(inPersonAppointmentsRow.children[1]).toHaveTextContent('31');
+    expect(inPersonAppointmentsRow.children[2]).toHaveTextContent('11');
+    expect(inPersonAppointmentsRow.children[3]).toHaveTextContent('1');
+    expect(inPersonAppointmentsRow.children[4]).toHaveTextContent('1');
+    expect(inPersonAppointmentsRow.children[4]).toHaveTextContent('1');
+
+    const followUpTextRow = rows[17];
+    expect(followUpTextRow.children[0]).toHaveTextContent('Text Message');
+    expect(followUpTextRow.children[1]).toHaveTextContent('38');
+    expect(followUpTextRow.children[2]).toHaveTextContent('18');
+    expect(followUpTextRow.children[3]).toHaveTextContent('8');
+    expect(followUpTextRow.children[4]).toHaveTextContent('8');
+    expect(followUpTextRow.children[5]).toHaveTextContent('18');
+
+    const appointmentsTotalRow = headers[3];
+    expect(appointmentsTotalRow.children[0]).toHaveTextContent('Appointments');
+    expect(appointmentsTotalRow.children[1]).toHaveTextContent('35');
+    expect(appointmentsTotalRow.children[2]).toHaveTextContent('15');
+    expect(appointmentsTotalRow.children[3]).toHaveTextContent('5');
+    expect(appointmentsTotalRow.children[4]).toHaveTextContent('5');
+    expect(appointmentsTotalRow.children[5]).toHaveTextContent('15');
+
+    const thankRow = rows[21];
+    expect(thankRow.children[0]).toHaveTextContent('Thank You Note');
+    expect(thankRow.children[1]).toHaveTextContent('33');
+    expect(thankRow.children[2]).toHaveTextContent('13');
+    expect(thankRow.children[3]).toHaveTextContent('3');
+    expect(thankRow.children[4]).toHaveTextContent('3');
+    expect(thankRow.children[5]).toHaveTextContent('13');
+
+    const initiationsTotalRow = headers[2];
+    expect(initiationsTotalRow.children[0]).toHaveTextContent('Initiations');
+    expect(initiationsTotalRow.children[1]).toHaveTextContent('36');
+    expect(initiationsTotalRow.children[2]).toHaveTextContent('16');
+    expect(initiationsTotalRow.children[3]).toHaveTextContent('6');
+    expect(initiationsTotalRow.children[4]).toHaveTextContent('6');
+    expect(initiationsTotalRow.children[5]).toHaveTextContent('16');
+
+    const referralsRow = rows[1];
```

Comment:
We can move this section up so that the order in the test matches the order on the page.